docs/user-guide/kubernetescurator.rst

```diff
@@ -385,3 +385,85 @@ To download data from your PVC, you can use the ``nemo-workspace-busybox`` Pod c
 
     # Copy directory in PVC /nemo-workspace/fizzbuzz to local file-system at $LOCAL_WORKSPACE/fizzbuzz
     kubectl exec nemo-workspace-busybox -- tar cf - /nemo-workspace/fizzbuzz | tar xf - -C $LOCAL_WORKSPACE
+
+
+Running existing Modules using Kubernetes Python client:
+
+------------------------------------------------------
+
+This approach involves programmatically establishing a connection to the scheduler pod in order to execute the current NeMo curator modules.
+
+.. code-block:: bash
+
+    # create a python environment
+    python3 -m venv venv
+    source venv/bin/activate
+    pip install kubernetes
+
+After installing Kubernetes Python client, you can run the following command:
+
+.. code-block:: bash
+
+    python3 examples/k8s/kubeclient.py --command "add_id --scheduler-address localhost:8786 --input-data-dir=/nemo-workspace/arxiv --output-data-dir=/nemo-workspace/arxiv-addid/" --kubeconfig "~/.kube/config"
+
+We can also use the same kubeclient.py to execute from a client pod. The client pod should be given necessary permiss
+ions to be able to exec into the scheduler pod and run the existing NeMo modules.
```

Comment:
1. Don't do a line break within a word, it messes up the formatting.
1. "existing NeMo modules" -> "existing NeMo Curator modules"